5/Português (Brasil).json

```diff
@@ -0,0 +1,52 @@
+[
+    {
+        "top":47,
+        "left":252,
+        "height":34,
+        "width":131,
+        "text": "Ei, Belfry..."
+    },
+    {
+        "top":53,
+        "left":435,
+        "height":64,
+        "width":110,
+        "text": "Não esquece de usar as"
+    },
+    {
+        "top":105,
+        "left":545,
+        "height":35,
+        "width":96,
+        "text": "*luvas térmicas!*"
+    },
+    {
+        "top":105,
+        "left":545,
+        "height":35,
+        "width":96,
+        "text": "*luvas térmicas!*"
+    },
```

Comment:
I see duplicate speech balloon entries here and in Page 6 below. Do you know if this is user error, or is this possibly caused by a bug in the transcript builder?

Reply:
oh oops yeah that's a mistake on my part. i forgot about the builder 😅

Reply:
Cool! Good to know! 👍 If you wouldn't mind fixing that on your end first, then I'll be happy to merge this PR.

Reply:
fixed :)